src/main/kotlin/calculator/parser/StringSplitModule.kt

```diff
@@ -0,0 +1,32 @@
+package calculator.parser
+
+class StringSplitModule {
+
+    private val regex = Regex("//(.)\n(.*)")
+
+    fun split(text: String): List<Int> {
+        val result = regex.find(text)
+        result?. let {
+            val customDelimiter = it.groupValues[1]
+            val numberList = it.groupValues[2].split(customDelimiter)
+            return numberList.map { str: String -> str.toPositiveNumber() }
+        }
+        return defaultDelimiter(text)
+    }
+
+    private fun defaultDelimiter(text: String): List<Int> {
+        return text.split(*DEFAULT_DELIMITER).map { it.toPositiveNumber() }
```

Comment:
```suggestion
        return text
            .split(*DEFAULT_DELIMITER)
            .map { it.toPositiveNumber() }
```
함수 체이닝 컨벤션을 적용하면 좋을 것 같아요